### PR TITLE
Feature: Support arm64 and new webpack source map diffs

### PR DESCRIPTION
**Pull request by briangann**
- refactored getPluginId to be a utility function that can be re-used
- implements changes needed to support source maps generated by new plugin-tools webpack configuration
- updated command line tools as needed to use new params

Note: this is a general fix for the current webpack configuration and is not specific to Yarn V3.

Fixes: #140

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -30,6 +30,22 @@
...
+      "cwd": "${workspaceFolder}",
+      "args": [
+        "-nonInteractive",
+        "y",
+        "-archiveUri",
+        "https://github.com/briangann/grafana-gauge-panel/releases/download/v2.0.0/briangann-gauge-panel-2.0.0.zip",
+        "-sourceCodeUri",
+        "https://github.com/briangann/grafana-gauge-panel/tree/v2.0.0",
+      ]
+    },
```
Comment (xnyo) [inline .vscode/launch.json:48]: Nit: maybe we can also take the yarn v3 source maps from this plugin and add a separate test case for it in `pkg/analysis/passes/jssourcemap/testdata` and `pkg/analysis/passes/jssourcemap/jssourcemap_test.go`, so this new case runs in CI as well?

**Review by xnyo — COMMENTED** [review]
Great work!! :smile: Leaving some comments:

File: pkg/sourcemap/sourcemap.go [hunk 2]
```diff
@@ -30,7 +31,8 @@ var ignoreStartingWith = []string{
 
 var replaceRegex = regexp.MustCompile(`^webpack:\/*`)
 
-func ParseSourceMapFromPath(sourceMapPath string) (*sourceMap, error) {
+func ParseSourceMapFromPath(pluginID string, sourceMapPath string) (*sourceMap, error) {
+	replaceRegex = regexp.MustCompile(fmt.Sprintf("^webpack://%s/*|^webpack:/*", pluginID))
```
[the remark this answers is not in the document]
Comment (briangann) [reply pkg/sourcemap/sourcemap.go:35]: always great feedback! i'm submitting these changes now

File: .vscode/launch.json (hunk 1 above)
Comment (briangann) [reply .vscode/launch.json:48]: i think that's a great idea, will implement it

**Comment by briangann** [comment]
think we're good to go here, changes have been applied and tests are passing

**Pull request merged**